spec.bs

```diff
@@ -670,6 +671,24 @@ button.
 documentPictureInPicture.requestWindow({ disallowReturnToOpener: true });
 </pre>
 
+## Disallow position reuse ## {#example-disallow-position-reuse}
+
+While a document picture-in-picture window is open, the user may choose
+to resize or reposition it. If the document picture-in-picture window is
+closed, then reopened later, the user agent may use the previous position
+and size as a hint for where to place the new window.
+
+The site can provide a hint to the user agent that reusing the previous
+document picture-in-picture window position and size is not desirable
+by setting the {{DocumentPictureInPictureOptions/disallowPositionReuse}}
+option.  For example, if the site is requesting the new document picture-in-
+picture window for an unrelated activity from the previous one, then the site
+could provide this hint to the user agent.
+
+<pre class="lang-javascript">
+documentPictureInPicture.requestWindow({ disallowPositionReuse: true });
```

Comment:
might be good to add width and height params into this example since theoretically a website caring about this will care about the size